easybuild/easyconfigs/b/BuildEnv/BuildEnv-1.0-intel-2015a-default.eb

```diff
@@ -0,0 +1,13 @@
+easyblock = "BuildEnv"
+
+name = 'BuildEnv'
+version = '1.0'
+versionsuffix = '-default'
```

Comment:
What kind of problems did you run into exactly?

Lmod does support 'meta' modules (which are versionless)...